tests/product/test_server_install.py

```diff
@@ -227,24 +227,6 @@ def test_install(self, installer=None):
             self.assert_has_default_config(container)
             self.assert_has_default_connector(container)
 
-    def test_install_worker_is_pa_master(self):
```

Comment:
ok.  I think it's good to test without java8_home set, but that's covered by all the other tests in this file.